### DISCUSSION TITLE
Data volume of Dependency Track

### DISCUSSION THREAD

**Discussion by aejazuddinam** [DependencyTrack/dependency-track#1215] [Q&A]
Hi,

After installation of dependency track in one of my docker container, we see the data volume is a local one mount on my docker server.
We need to get the data volume onto one of the NFS partitions so it gets some backup. Please advise.

Regards,
Aejaz

**Comment by stevespringett** [comment]
Refer to the docs: https://docs.dependencytrack.org/getting-started/deploy-docker/
